src/components/teas.json

```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "id": 1,
+    "tea_name": "Vietnamese Oolong",
```

Comment:
Da habe ich in Erinnerung, dass Anderas mal gesagt hat, wenn es hier eigentlich eh um Tees geht und du schreibst nochmal `tea_name` ist es unnötige Doppelung. Du kannst also einfach nur `name` nehmen.

Ansonsten glaube ich, ist es eher Standard camelCase bei den Objekten zu verwenden, damit du die dann auch so entsprechend im Code ansprechen kannst. Wäre bei dir dann noch der `point_purchase` => `pointPurchase`